src/Symfony/Component/Console/Attribute/Argument.php

```diff
@@ -99,6 +99,6 @@ public function toInputArgument(): InputArgument
      */
     public function resolveValue(InputInterface $input): mixed
     {
-        return $input->hasArgument($this->name) ? $input->getArgument($this->name) : null;
+        return $input->getArgument($this->name);
```

Comment:
removed `$input->hasArgument($this->name)` as it's better to get the "argument does not exist" exception than a TypeError or just null (if nullable)